master/buildbot/test/fake/fakedb.py

```diff
@@ -528,6 +533,7 @@ class LogChunk(Row):
         compressed=0)
 
     required_columns = ('logid', )
+    utf8_columns = ('content', )
```

Comment:
I am not sure exactly what this is about. fakedb is about faking the db. I dont see how this code will help fixing pg